cases/models.py

```diff
@@ -367,6 +370,15 @@ def _timeline(self, actions, action_fn, complaints, history_to_show, user=None):
                 "summary": action_fn(action),
                 "action": action,
             }
+            row["files"] = []
+            for _file in action.files.all():
```

Comment:
This will perform a db query per action, I think - we should probably make sure that files are included in a relevant `prefetch_related` so that they're prefetched at the time - I guess in actions_reversed, or perhaps even here in _timeline depending upon history_to_show.